modules/core/src/test/java/org/apache/ignite/internal/processors/metastorage/DistributedMetaStorageTest.java

```diff
@@ -151,6 +158,46 @@ public void testSingleNode() throws Exception {
         }
     }
 
+    /**
+     * Test verifies that Distributed Metastorage on client yields error if client is not connected to some cluster.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testDistributedMetastorageOperationsOnClient() throws Exception {
+        String clientName = "client0";
+
+        String key = "key";
+        String value = "value";
+
+        GridTestUtils.runAsync(() -> startGrid(clientName));
```

Comment:
U can use **GridAbstractTest#startClientGrid(int)**

Reply:
But I need the name to later get the semi-started node with `IgnitionEx.gridx`